solc/CommandLineInterface.cpp

```diff
@@ -1195,6 +1198,10 @@ void CommandLineInterface::handleCModelHeaders(
 		cov.print(_os);
 	}
 	for (auto const& ast : _asts)
+	{
+		modelcheck::MainFunction_1(*ast, _con, true).print(_os);
```

Comment:
I may be mistaken, but I thought we were going for something along the lines of,
1. A header/source pair for the contract code
2. A header/source pair for the runtime stuff (like callstate)
3. One or more models (`mains`) that would be linked against this with `Seahorn` to verify properties
4. Some kind of interactive execution (we haven't started this yet)
I suppose we can update this is a later commit. I need refine my part of the generator before it's able to make files that are linkable ootb. Just wanted to make sure we were heading in the same direction. :)